server/odc-service/src/main/java/com/oceanbase/odc/service/iam/UserPermissionService.java

```diff
@@ -150,6 +150,7 @@ public List<UserPermissionResp> list(@NonNull String resourceIdentifier) {
                 .collect(Collectors.toList());
     }
 
+    @Deprecated
```

Comment:
shall we delete these deprecated methods?

Reply:
Yes, I confirmed it with font-end and deleted these codes.